assemble/conf/accumulo-env.sh

```diff
@@ -73,11 +73,13 @@ export CLASSPATH
 
 ## JVM options set for all processes. Extra options can be passed in by setting ACCUMULO_JAVA_OPTS to an array of options.
 read -r -a accumulo_initial_opts < <(echo "$ACCUMULO_JAVA_OPTS")
-JAVA_OPTS=("${accumulo_initial_opts[@]}"
+JAVA_OPTS=(
   '-XX:OnOutOfMemoryError=kill -9 %p'
   '-XX:-OmitStackTraceInFastThrow'
   '-Djava.net.preferIPv4Stack=true'
-  "-Daccumulo.native.lib.path=${lib}/native")
+  "-Daccumulo.native.lib.path=${lib}/native"
+  "${accumulo_initial_opts[@]}"
```

Comment:
This is interesting moving this last, will make the ACCUMULO_JAVA_OPTS absolute last like you mentioned in the description.  Makes wonder if the -D options can be overridden.   If they can this change will facilitate it.

Reply:
If they can't, then this change fixes one use case, but harms another. We'd need to decide which is a better default. In the end, this is a user configurable file, and it's really up to the user to make sure their config is doing what they need. So, I'm not too worried about it.